src/vconsole/vconsole-setup.c

```diff
@@ -190,11 +193,12 @@ static int font_load_and_wait(const char *vc, const char *font, const char *map,
  * font. It also allows to restart systemd-vconsole-setup.service,
  * to apply a new font to all VTs.
  */
-static void font_copy_to_all_vcs(int fd) {
+static void font_copy_to_all_vcs(int fd, unsigned int vc_cnt) {
         struct vt_stat vcs = {};
         struct unimapdesc unimapd;
         _cleanup_free_ struct unipair* unipairs = NULL;
-        int i, r;
+        unsigned int i;
```

Comment:
not tht it mtters at all but we write just "unsigned" instead of "unsigned int" pretty much everywhere else, so I figure we should here too